cmd/env/command.go

```diff
@@ -40,8 +40,7 @@ their names to upper case and replacing punctuation with underscores. For exampl
       --cassandra.connections-per-host    CASSANDRA_CONNECTIONS_PER_HOST
       --metrics-backend                   METRICS_BACKEND
 
-The following configuration options are only available via environment variables:
-`)
+The following configuration options are only available via environment variables:`)
```

Comment:
why? The formatting is better with a blank line.
```
$ go run ./cmd/query/main.go env

All command line options can be provided via environment variables by converting
their names to upper case and replacing punctuation with underscores. For example,

      command line option                 environment variable
      ------------------------------------------------------------------
      --cassandra.connections-per-host    CASSANDRA_CONNECTIONS_PER_HOST
      --metrics-backend                   METRICS_BACKEND

The following configuration options are only available via environment variables:

      SPAN_STORAGE_TYPE string         The type of backend (cassandra, elasticsearch, memory) used for trace storage. (default "cassandra")
      DEPENDENCY_STORAGE_TYPE string   The type of backend used for service dependencies storage. (default "${SPAN_STORAGE}")
```

Reply:
then I would add `+ "\n"` before paren. Not having new-line makes display less readable